gradle.properties

```diff
@@ -5,16 +5,15 @@ group=org.jetbrains.kotlinx
 version=0.4.1
 versionSuffix=SNAPSHOT
 
-defaultKotlinVersion=1.8.21
+defaultKotlinVersion=1.9.20
 dokkaVersion=1.8.20
 serializationVersion=1.5.1
 
 java.mainToolchainVersion=8
 java.modularToolchainVersion=11
 
 kotlin.mpp.stability.nowarn=true
```

Comment:
Probably this one could be removed now

Reply:
Addressed in ca5725b219549e323bfb62f3fb143c6d6cc98b8f.